malariagen_data/ag3.py

```diff
@@ -1143,14 +1169,13 @@ def snp_calls(
                 [
                     xarray.concat(
                         [
-                            self._snp_calls_dataset(
+                            self.snp_calls(
```

Comment:
Hi @alimanfoo, I'm not completely clear why we no longer need to call `_snp_calls_dataset` here?

Reply:
Calling recursively like this allows multiple releases to be provided as the `sample_sets` argument. E.g., `sample_sets = ['v3', 'v3.1', 'v3.2']`.

Reply:
An alternative approach could be to properly flatten out the `sample_sets` parameter within the `_prep_sample_sets_arg` function, i.e., turn something like `['v3', 'v3.1', 'v3.2']` into a list of sample sets. I might look into that.